common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -210,17 +214,17 @@ def test_submit_blank_answer(self):
             lambda: self.problem_page.problem_name == self.problem_name,
             "Make sure the correct problem is on the page"
         )
-        # Leave the problem unchanged and click check.
-        self.assertNotIn('is-disabled', self.problem_page.q(css='div.problem button.check').attrs('class')[0])
-        self.problem_page.click_check()
+        # Leave the problem unchanged and click submit.
+        self.assertNotIn('is-disabled', self.problem_page.q(css='.problem .submit').attrs('class')[0])
```

Comment:
Then we need to update this test or split it further.

We can check if it is not disabled then attempt a submit and expect and incorrect answer.